pages/ghg/ghg.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: GHG Alignment
+permalink: /ghg
+---
+
+# GHG Alignment
+
+The Greenhouse Gas (GHG) Protocol is a widely used international accounting tool for governments and businesses to understand, quantify, and manage their greenhouse gas emissions. [Read more about the GHG Protocol](resources/glossary#greenhouse-gas-ghg-protocols).
+
+The Tech Carbon Standard (TCS) specifically focuses on the technology landscape, a subset of a company's overall GHG emissions. This focus is to help organisations identify technology related sources of GHG emissions. Furthermore, the TCS provides guidance for selecting the calculation approaches, reliable sources of data and emissions factors to be able to calculate those emissions, which can then be used to roll-up to corporate level GHG reporting.
+
+<div class="grid place-content-center">
+  <img class="max-h-72 place-self-center" src="/assets/images/ghg-emissions-process.png" title="Steps in identifying and calculating GHG emissions" />
+  <a class="text-center" href="https://ghgprotocol.org/sites/default/files/standards/ghg-protocol-revised.pdf">Steps in identifying and calculating GHG emission - GHG Protocol Reporting Standard</a>
+</div>
+
+The TCS is most helpful and applicable for identifying emissions sources and calculating GHG Scope 3 emissions. Scopes 1 and 2 should never be double counted, and these Scopes are often accounted for at corporate level by taking consideration of the energy and fuels purchased by the company. So, we would advise not to use the TCS categories for reporting GHG Scopes 1 and 2, to avoid the risk of double counting. For example; the energy consumption data collected by the company does not take into any consideration of how that energy is used or distributed between heating, lighting, and technology use. These categories should be reserved for breakdown and analysis of it's contributing factors to Scope 1 and 2, rather than reporting them. GHG Protocol is focuses on an attributional model, meaning that the identification of sources and the emissions calculations are used for accoutning and reporting, "attributing" emissions to the company, fitting squarely into Scope 1, 2 or 3. But where the TCS is used to breakdown and analyse these emissions for the purpose of influencing and reducing the company's GHG emissions, the TCS shifts to a consequential model. The detailed breakdown and understanding of the sources of emissions enables companies to identify the potential effects of making changes to mitigate and reduce GHG emissions.
```

Comment:
```suggestion
The TCS is most helpful and applicable for identifying emissions sources and calculating GHG Scope 3 emissions. Scopes 1 and 2 should never be double counted, and these Scopes are often accounted for at corporate level by taking consideration of the energy and fuels purchased by the company. So, we would advise not using the TCS categories for reporting GHG Scopes 1 and 2, to avoid the risk of double counting. For example, the energy consumption data collected by the company does not take any consideration of how that energy is used or distributed between heating, lighting, and technology use. These categories should be reserved for breakdown and analysis of their contributing factors to Scope 1 and 2, rather than reporting them. The GHG Protocol focuses on an attributional model, meaning that the identification of sources and the emissions calculations are used for accounting and reporting, "attributing" emissions to the company, fitting squarely into Scope 1, 2 or 3. But since the TCS is used to breakdown and analyse these emissions for the purpose of influencing and reducing the company's GHG emissions, the TCS shifts to a consequential model. This detailed breakdown and understanding of the sources of emissions enables companies to identify the potential effects of making changes to mitigate and reduce GHG emissions.
```